### PR TITLE
Lambda.part1.exercise2(Musatov)

### PR DESCRIPTION
**Pull request by andremus32**

### CODE REVIEW

File: src/test/java/lambda/part1/exercise/Exercise2.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        for (int i = 0; i < number; ++i) {
+            joiner.add(string);
+        }
+        String result = joiner.toString();
+        return result.equals(delimiter) ? "" : result;
+    }
+
+    @Test
+    public void implementsStringMultiplierUsingObjectMethodReference() {
+        Multiplier<String> multiplier = null;
```
Comment (EKrysenko) [inline src/test/java/lambda/part1/exercise/Exercise2.java]: Isn`t it better to use in-line initialization of Multiplier?

File: src/test/java/lambda/part1/exercise/Exercise2.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+        Exercise2 exercise2 = new Exercise2();
```
Comment (EKrysenko) [inline src/test/java/lambda/part1/exercise/Exercise2.java]: You can use "this." instead of Exercise2 instance creating.

Comment (andremus32) [reply src/test/java/lambda/part1/exercise/Exercise2.java]: И то верно.

File: src/test/java/lambda/part1/exercise/Exercise2.java (hunk 1 above)
Comment (andremus32) [reply src/test/java/lambda/part1/exercise/Exercise2.java]: Fixed.

**Pull request merged**